samples/chaincode/cc-tools-demo/README.md

```diff
@@ -0,0 +1,274 @@
+# CC-Tools-Demo Tutorial
+
+This tutorial shows how to build, install and test a Go Chaincode developed using the [CC-Tools]() framework and integrating it with the Fabric Private Chaincode (FPC) framework.
+
+This tutorial illustrates a simple use case where we follow the [cc-tools-demo]() chaincode which is based on standard fabric and then convert it to an FPC chaincode achieving FPC security capabilities.
+
+This tutorial is based on the [FPC with CC-Tools integration project]() and all our design choices are explained here in the [design document]()
+Here are the steps to accomplish this:
+
+* Clone and copy the cc-tools-demo chaincode
+* Edit the chaincode to became an FPC chaincode instead of normal fabric
+* Build your FPC Go Chaincode
+* Launch a Fabric network
+* Install and instantiate your chaincode
+* Invoke transactions by using the FPC simple-cli
+
+## Prerequisites
+
+This tutorial presumes that you have installed FPC on your `$GOPATH` as described in the FPC [README.md](../../../README.md#clone-fabric-private-chaincode) and `$FPC_PATH` is set accordingly.
+
+We also need a working FPC development environment. As described in the "Setup your Development Environment" Section of the FPC [README.md](../../../README.md#setup-your-development-environment), you can use our docker-based dev environment (Option 1) or setup your local development environment (Option 2).
+We recommend using the docker-based development environment and continue this tutorial within the dev container terminal.
+
+Moreover, within your FPC development you have already installed the FPC Go Chaincode Support components.
+See the installation steps in [ecc_go/README.md](../../../ecc_go/README.md#installation).
+
+We also assume that you are familiar with Fabric chaincode development in go.
+Most of the steps in this tutorial follow the normal Fabric chaincode development process, however, there are a few differences that we will highlight here.
+
+## Clone and copy the cc-tools-demo chaincode
+
+Clone the [cc-tools-demo]() repository and copy the [chaincode]() folder. Then paste it in the root directory for cc-tools-demo here.
+
+```bash
+cd ~
+git clone https://github.com/hyperledger-labs/cc-tools-demo.git
+cp -a ~/cc-tools-demo/chaincode/. $FPC_PATH/samples/chaincode/cc-tools-demo/
+cd $FPC_PATH/samples/chaincode/cc-tools-demo
+```
```

Comment:
Can we just clone directly?
```
git clone https://github.com/hyperledger-labs/cc-tools-demo.git $FPC_PATH/samples/chaincode/cc-tools-demo
```

Reply:
We don't need the whole cc-tools-demo repo, I only want the chaincode as the repo includes multiple other files

Reply:
Done using sparse checkout